keyboards/crkbd/keymaps/oo/config.h

```diff
@@ -0,0 +1,28 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define SPLIT_USB_DETECT
```

Comment:
Is this required for your board?

Reply:
Hi @drashna, this is a good question. I remember trying a few configurations earlier in the process of setting up this board but I couldn't remember if I'd specifically tested all the options so I reviewed the docs in the [Split Keyboard](https://docs.qmk.fm/#/feature_split_keyboard?id=hardware-configuration-options) section.

It indicates that the rules.mk file can contain ```SPLIT_KEYBOARD = yes``` or the config.h can contain ```#define SPLIT_USB_DETECT```. Further down the page it does indicate that the config.h define may also be optional for Pro Micro boards (which I am using for both board halves).

I went ahead and tested all the scenarios. Using neither option or just the rules.mk statement both result in the right side board thinking it is a master (instead of a slave) even when the usb is plugged into the left. But when I have the define in config.h, that does result in the right side board properly operating as a slave to the left side.

tl;dr, yes, this is required for my board.